toolset/setup/linux/languages/rbx-2.5.sh

```diff
@@ -12,6 +12,9 @@ RETCODE=$(fw_exists ${IROOT}/rbx-2.5.installed)
 # update this when it changes
 RBX_VERSION=2.5.2
 
+# This version of RBX needs clang 3.4
+sudo apt-get -yq --force-yes install clang-3.4
```

Comment:
Ugh... more apt-get, more sudo.

@nbrady-techempower if you have time, can you quickly experiment to see if you can make `clang-3.4` into its own systool dependency and test out `fw_apt_to_iroot`? I would love to move away from as many of these apt-get as possible.